tools/mass_perldoc_md_creation.pl

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/perl -w
```

Comment:
I would be in favour of simplifying the script. I would get rid of `@script_list`: no need to explicitly list (and maintain) the list of scripts/modules when a Unix `find` command can do this for you. What do you think?

Reply:
I would tend to agree with @cmadjar simply because we do not have all the files perl documented properly.
It is a bit like Travis where we add the .js files to run npm compile and phpcs on the php scripts; but those are ran on files manually added... so it gives us flexibility to exclude some if need be.